src/main/java/Task.java

```diff
@@ -0,0 +1,83 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected String taskLabel;
+
+    static final int TASK_NUMBER_OFFSET = 1;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /** Mark done task with X */
+    public String getStatusIcon() {
+        return (isDone ? "[X]" : "[ ]");
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(boolean done) {
+
+        isDone = done;
+    }
+
+    /**
+     * executeNonAdd method executes all instructions
+     * except the instructions to add tasks to the list
+     */
+    public static void executeNonAdd(String command, Task[] listOfTasks, int currentNumber) {
+        System.out.print("    ____________________________________________________________\n");
+        if (command.equals("bye")) {
+            System.out.println("    Bye. Hope to see you again soon!");
+        } else if (command.equals("list")) {
+            list(listOfTasks, currentNumber);
```

Comment:
do you think implementing "bye" and "list" in Duke.java is a better option so you dont have to pass the Task[] around?